### ISSUE TITLE
Mac menu bar

### ISSUE THREAD

**Issue by matthewmaier** [matthewmaier/howstr#3]
When the app is full screen on a Mac the Mac's menu bar covers up the icons (because Mac).

**Comment by jmwright** [comment]
Why is it again that you're doing custom window controls?

**Comment by matthewmaier** [comment]
One consistent interface across all systems.

**Comment by jmwright** [comment]
Are you doing it for the users? Users that cross between systems are
already familiar with the differences in window controls. I run Ubuntu
Linux as my primary, but have to use Windows a lot for work. I'm
comfortable with the differences, and it doesn't alter my ability to use
Howstr.

**Comment by matthewmaier** [comment]
I don't have any particular attachment to custom window controls themselves. Those were just the first controls I created. This particular issue wouldn't be helped by getting rid of the custom window controls anyway. The Mac menu bar is covering up lots of other controls too.

**Comment by jmwright** [comment]
Yeah, I was just using this issue as an opportunity to take a step back and re-evaluate assumptions.
